src/components/sidebar/helpers/add-nav-item-meta-data.ts

```diff
@@ -12,6 +12,27 @@ interface AddNavItemMetaDataResult {
 	itemsWithMetadata: NavItemWithMetaData[]
 }
 
+/**
+ * Returns an object for rendering a Badge in a sidebar item if a <sup> tag is
+ * founnd in its title.
+ */
+const getBadgeFromTitle = (title: string) => {
+	let badge = null
+
+	const regex = new RegExp(/<sup>(.*)<\/sup>$/)
```

Comment:
```suggestion
	const regex = new RegExp(/^<sup>(.*)<\/sup>$/)
```
Should the use always start with `<sup>`?

Reply:
Nope, the titles don't need to start with `<sup>`! Here's an example of the usage (and I'll add one to the comment I'm writing): https://github.com/hashicorp/cloud.hashicorp.com/blob/33c65ea6659a234b13ce70ce8fc30d82a3e53383/data/docs-nav-data.json#L158